libnopegl/src/node_texture.c

```diff
@@ -371,6 +372,43 @@ static int texture_prefetch(struct ngl_node *node)
     ngli_image_init(&s->image, &image_params, &s->texture);
     s->image.rev = s->image_rev++;
 
+    if (s->rtt) {
+        /* Transform the color textures coordinates so it matches how the
+         * graphics context uv coordinate system works regarding render targets */
+        ngli_gpu_ctx_get_rendertarget_uvcoord_matrix(gpu_ctx, s->image.coordinates_matrix);
+
+        int depth_format = NGLI_FORMAT_UNDEFINED;
+        if (s->renderpass_info.features & NGLI_RENDERPASS_FEATURE_STENCIL)
+            depth_format = ngli_gpu_ctx_get_preferred_depth_stencil_format(gpu_ctx);
+        else if (s->renderpass_info.features & NGLI_RENDERPASS_FEATURE_DEPTH)
+            depth_format = ngli_gpu_ctx_get_preferred_depth_format(gpu_ctx);
+
+        const struct rtt_params rtt_params = {
+            .width = params->width,
+            .height = params->height,
+            .nb_interruptions = s->renderpass_info.nb_interruptions,
+            .nb_colors = 1,
+            .colors[0] = {
+                .attachment = s->texture,
+                .load_op = NGLI_LOAD_OP_CLEAR,
+                .store_op = NGLI_STORE_OP_STORE,
+                .clear_value[0] = o->clear_color[0],
+                .clear_value[1] = o->clear_color[1],
+                .clear_value[2] = o->clear_color[2],
+                .clear_value[3] = o->clear_color[3],
```

Comment:
not possible to use `NGLI_ARG_VEC4()`?

Reply:
Nope. But this will change in another PR where I introduce a dedicated clear_color structure.